modules/creating-infra-machines.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/cluster-tasks.adoc
+
+[id="creating-infra-machines_{context}"]
+= Creating infra machines
+
+if you need infra machines to have dedicated configurations, then you must create an infra pool.
+
+.Procedure
+
+. Create a `MachineConfigPool` that contains both the worker role and your custom role as `MachineConfig` selector:
++
+[source,terminal]
+----
+$ cat infra.mcp.yaml
```

Comment:
Is `cat` a sufficient command here? All it does is show the contents of the file. I've seen other places either say something like "Create a file with the following contents", or they use `cat` in combination with <<EOF, etc. to actually insert the contents into the file.